third-party/linshare/src/main/java/org/apache/james/linshare/LinshareBlobExportMechanism.java

```diff
@@ -55,31 +54,29 @@
     public ShareeStage blobId(BlobId blobId) {
         return mailAddress -> explanation -> fileCustomPrefix -> fileExtension -> () ->  {
```

Comment:
Do we still need `explanation` here ?

Reply:
Yes, we do not use value of it but the design of this.

Reply:
More precisely, as the file is directly uploaded to the user "mySpace" we can make no use of this 'explanation'.

The component providing the explanation to the end user will no longer be James by the intermediate of LinShare but OpenPaaS directly.